azure-pipelines.yml

```diff
@@ -45,255 +35,214 @@ resources:
+          helixRepo: dotnet/arcade-validation
+          jobs:
+          - job: Windows_NT
 
-  # Jobs that should only run as part of internal builds.
-      - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
-        - job: Validate_Signing
-          pool:
-            name: $(PoolProvider) # This is a queue-time parameter; Public default is NetCorePublic-Pool; Internal default is NetCoreInternal-Pool
-            demands: ImageOverride -equals windows.vs2019.amd64
-          strategy:
-            matrix:
-              Test_Signing:
-                _BuildConfig: Debug
-                _SignType: test
-              Real_Signing:
-                _BuildConfig: Release
-                _SignType: real
-          steps:
+            variables:
+            - _BuildConfig: Release
+            - _SignType: real
+            - _InternalBuildArgs: /p:DotNetSignType=$(_SignType)
+                /p:TeamName=$(_TeamName)
+                /p:DotNetPublishUsingPipelines=$(_PublishUsingPipelines)
+                /p:OfficialBuildId=$(BUILD.BUILDNUMBER)
+                
+            steps:
             - checkout: self
               clean: true
-            - task: CopyFiles@2
-              displayName: Copy test packages to artifacts directory
+            - task: PowerShell@2
+              displayName: Setup Private Feeds Credentials
+              condition: eq(variables['Agent.OS'], 'Windows_NT')
               inputs:
-                sourceFolder: $(Build.SourcesDirectory)\src\validation\resources
-                targetFolder: $(Build.SourcesDirectory)\artifacts\packages\$(_BuildConfig)\NonShipping
-            - powershell: eng\common\build.ps1
+                filePath: $(Build.SourcesDirectory)/eng/common/SetupNugetSources.ps1
+                arguments: -ConfigFile $(Build.SourcesDirectory)/NuGet.config -Password $Env:Token
+              env:
+                Token: $(dn-bot-dnceng-artifact-feeds-rw)
+            - script: eng\common\cibuild.cmd
                 -configuration $(_BuildConfig)
-                -restore
                 -prepareMachine
-                -sign
-                -ci
-                /p:DotNetSignType=$(_SignType)
-                /p:TeamName=DotNetCore
-                /p:OfficialBuildId=$(BUILD.BUILDNUMBER)
-              displayName: Sign packages
+                $(_InternalBuildArgs)
+              displayName: Windows Build / Publish
+            - task: ComponentGovernanceComponentDetection@0
+              displayName: Component Governance scan
 
-- ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
-  - template: eng\common\templates\post-build\post-build.yml
-    parameters:
-      publishingInfraVersion: 3
-      # Symbol validation isn't being very reliable lately. This should be enabled back
-      # once this issue is resolved: https://github.com/dotnet/arcade/issues/2871
```

Comment:
nit: restore comment

Reply:
the issue mentioned on the comment is now resolved (since 2019)